docs/framework/whats-new/obsolete-members.md

```diff
@@ -710,8 +710,8 @@ The tables in this article list the type members that are obsolete in the .NET F
   
 |Type|Member|Message|  
 |----------|------------|-------------|  
-|<xref:System.Xml.ValidationType?displayProperty=nameWithType>|<xref:System.Xml.ValidationType>|Validation type should be specified as <xref:System.Xml.ValidationType?displayProperty=nameWithType> or <xref:System.Xml.ValidationType?displayProperty=nameWithType>.|  
-|<xref:System.Xml.ValidationType?displayProperty=nameWithType>|<xref:System.Xml.ValidationType>|XDR validation through <xref:System.Xml.XmlValidatingReader?displayProperty=nameWithType> is obsoleted.|  
+|<xref:System.Xml.ValidationType?displayProperty=nameWithType>|<xref:System.Xml.ValidationType.Auto>|Validation type should be specified as <xref:System.Xml.ValidationType?displayProperty=nameWithType> or <xref:System.Xml.ValidationType?displayProperty=nameWithType>.|  
```

Comment:
3rd xref link should be System.Xml.ValidationType.DTD and 4th link should be System.Xml.ValidationType.Schema